packages/nextjs-mf/src/internal.ts

```diff
@@ -140,16 +140,24 @@ export const generateRemoteTemplate = (url: string, global: any) => `new Promise
             return true
           }
         }
-        try {
-          ${global}.init(new Proxy(shareScope, handler))
-        } catch (e) {
+        if (!${global}.__initialized) {
+          try {
+            ${global}.init(new Proxy(shareScope, handler), initScope)
+          } catch (e) {
 
+          }
+          ${global}.__initialized = true
         }
-        ${global}.__initialized = true
       }
     }
     if (!${global}.__initialized) {
+    try {
```

Comment:
If remote wasn't initialized. Manually call it. 
General pattern Needed for various backward compatibility reasons